inc/admin.php

```diff
@@ -70,14 +71,16 @@ function nkdeletepending_options_page() {
 					*/
 					global $wpdb;
 					$wpdb->query( 
-						$wpdb->prepare( "DELETE FROM $wpdb->comments WHERE comment_approved = 0" )
+						$wpdb->prepare( "DELETE FROM $wpdb->comments WHERE comment_approved = %d", 0 )
 					);
-					$deleted_comment_ids = wp_list_pluck( $comments, 'comment_ID' );
-					clean_comment_cache( $deleted_comment_ids );
-					foreach ( $deleted_comment_ids as $comment_id ) {
+					clean_comment_cache( $pending_comment_ids );
+					foreach ( $pending_comment_ids as $comment_id ) {
 						do_action( 'wp_set_comment_status', $comment_id, 'delete' );
 					}
 
+					$pending_comments_count = get_comments( 'status=hold&count=true' );
+					error_log( $pending_comments_count );
```

Comment:
Remove `error_log` and other debug statements.

Reply:
Fixed.